user/details/index.html

```diff
@@ -23,25 +24,49 @@ <h1>User Management</h1>
           <li class="accordion-academic">
             <div class="visible-content">
               <h2>Professional Details</h2>
-              <img src="../images/arrow-icon.svg" alt="expand" />
+              <div class="tool-parent">
```

Comment:
can the class name be something more meaningful

Reply:
done